src/index.html

```diff
@@ -7,8 +7,34 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link 
+      rel="preconnect" 
+      href="https://fonts.gstatic.com"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <a href="#home">
+        <img 
+        src="./images/logo.png" 
+        alt="icon Moyo" 
+        class="img"
+        >
+      </a>
+      <nav class="navigation">
+        <ul class="nav_list">
+          <li><a href="#" class="is-active">APPLE</a></li>
+          <li><a href="#">SAMSUNG</a></li>
```

Comment:
```suggestion
          <li class="nav__item">
            <a class="nav__link" href="#apple" class="is-active">APPLE</a>
          </li>

          <li class="nav__item">
            <a class="nav__link" href="#samsung">SAMSUNG</a>
          </li>
```

Fix everywhere, add blank line between multiple sibling blocks

Reply:
isn't fixed yet